ovs/extensions/db/arakoon/ArakoonInstaller.py

```diff
@@ -187,6 +190,26 @@ def __init__(self):
         raise RuntimeError('ArakoonInstaller is a complete static helper class')
 
     @staticmethod
+    def get_etcd_config_path():
+        return ArakoonInstaller.ETCD_CONFIG_PATH
```

Comment:
What's the purpose of these methods? You could just simply get or set the property itself. And if you still need a setter and getter, it might be a good idea to actually use setters and getters.